packages/interfaces/CHANGELOG.md

```diff
@@ -5,6 +5,24 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
```

Comment:
This disclaimer should be removed as we don't follow the _keep a changelog_ format any more

Reply:
Addressed it here + added missing CHANGELOG files
https://github.com/waku-org/js-waku/pull/1178